src/qtism/data/storage/xml/marshalling/OutcomeDeclarationMarshaller.php

```diff
@@ -126,7 +143,10 @@ protected function unmarshall(DOMElement $element)
             }
 
             // deal with views.
-            if (Version::compare($version, '2.1.0', '>=') === true && ($views = $this->getDOMElementAttributeAs($element, 'view')) !== null) {
+            if (Version::compare($version, '2.1.0', '>=') === true && ($views = $this->getDOMElementAttributeAs(
+                    $element,
+                    'view'
+                )) !== null) {
```

Comment:
could you provide here proper multiline if statement as suggested earlier or revert that place